Formula/grib-api.rb

```diff
@@ -25,6 +26,7 @@ def install
     mkdir "build" do
       args = std_cmake_args
       args << "-DBUILD_SHARED_LIBS=OFF" if build.with? "static"
+      args << "-DPNG_PNG_INCLUDE_DIR=#{Formula["libpng"].opt_include}/include" << "-DENABLE_PNG=ON" if build.with? "libpng"
```

Comment:
Sorry if I wasn't clear here. You don't need the extra `/include` when calling `opt_include`.